lib/components/logIn.dart

```diff
@@ -33,8 +44,36 @@ class _LogInScreenState extends State<LogInScreen> {
     super.dispose();
   }
 
+  Future uploadProfilePic(BuildContext context) async {
```

Comment:
would be great if you moved this upload logic too into a service file, like `FirebaseStorageService.dart`

make this a singleton usage too for now.

Reply:
Well i've an suggestion for that since this method of uploading image to firebase storage bucket and then retrieving takes too much time .
So i would like to suggest we should rather save the imageurl in the firebase cloudstore documents where the user info is saved and we will define each user by docid =uid so it will be easy that way and also this method would be fast
I've worked on this before so if u'll like this feature i can do this in an another issue?

Reply:
Yes please go ahead, that is the right way.